python/lsst/ts/MTAirCompressor/aircompressor_csc.py

```diff
@@ -241,6 +244,7 @@ async def disconnect(self):
 
     async def end_start(self, data):
         """Enables communication with the compressor."""
+        print("end start 1")
```

Comment:
Consider using `self.log.debug` instead...

Reply:
That's a leftover from debugging..didn't notice it. Also all others. Print statements were removed.